py/float.go

```diff
@@ -398,14 +398,14 @@ func (a Float) M__ge__(other Object) (Object, error) {
 func init() {
 	FloatType.Dict["is_integer"] = MustNewMethod("is_integer", func(self Object) (Object, error) {
 		if a, ok := convertToFloat(self); ok {
-			f, err := FloatAsFloat64(a)
-			if err != nil {
-				return nil, err
+			f, _ := FloatAsFloat64(a)
+			if math.Floor(f) == f {
+				return True, nil
 			}
-			b := math.Abs(f - math.Round(f))
-			return NewBool(b < math.SmallestNonzeroFloat64), nil;
+			return False, nil
 		}
-		return nil, AttributeError
+		_, e := cantConvert(self, "float")
```

Comment:
Please return cantConvert directly

Reply:
How do I use the second return value directly?

Sorry, I'm looking for that. But I can't find any information.